.circleci/config.yml

```diff
@@ -0,0 +1,12 @@
+version: 2.1
+
+jobs:
+  build:
+    docker:
+      - image: public.ecr.aws/l8e3r2a5/rmx-ci-standard:latest
+    steps:
+      - checkout
+      - run: cd $HOME && git clone git@github.com:figly/ocaml-scripts
```

Comment:
you can drop this line, the `rmx-ci-standard` image includes the `ocaml` dependencies already